src/app/shared/services/dia-backend/asset/dia-backend-asset-repository.service.ts

```diff
@@ -73,15 +95,9 @@ export class DiaBackendAssetRepository {
     return this.list$({ offset, limit, isOriginalOwner: true });
   }
 
-  getPostCaptures$(options?: { limit?: number; offset?: number }) {
-    return iif(
-      () => options?.limit !== undefined,
-      this.list$({
-        isOriginalOwner: false,
-        orderBy: 'source_transaction',
-        limit: options?.limit,
-        offset: options?.offset,
-      }),
+  getPostCaptures$() {
+    return merge(
```

Comment:
The order of emissions from input observables are not enforced, therefore it's theoretically possible that the cache emits after the API response emits and that wouldn't be an expected behavior. Since it's very unlikely to happen in real life and even if it happens it won't be a critical issue, I won't request change for it.

Reply:
I don't think this will happen. Since the cache will be updated by [`tap(response => this.postCapturesCache$.next(response))*`](https://github.com/numbersprotocol/capture-lite/pull/600/files/b5d20f2bafb1a83df82e1c7955b5f218a231ccd6#diff-70f3e585f461105d08f253b8ed0b479278d855b905c21ea263f5c24418bc618eR110). Thus, the cache will always emit the latest value after the data is fetched from the backend.